umh-core/Makefile

```diff
@@ -16,14 +16,16 @@
 
 IMAGE_NAME = umh-core
 TAG = latest
+# Dynamically select podman if available, otherwise use docker
+OCI_RUNTIME := $(shell command -v podman >/dev/null 2>&1 && echo podman || echo docker)
```

Comment:
are we sure we want this? the advantage of a devcontainer is, that we standardize the development environment. i can already see how this could cause to future issues, because some people work in docker, and some work in podman, so maybe some make commands will only run in one of those.

if everyone introduces his own slight adjustments to his local environment, then it will not become clear for a new person what he needs to do to set it up.

Reply:
we can also use podman, but then we need to adjust the devcontainer to work with podman as well

Reply:
can we take this out for now and discuss this evening?